### PR TITLE
[#347] Upgrade rollbar gem

### PR DESCRIPTION
**Pull request by ruslankhaertdinov**
Related to: https://github.com/fs/rails-base/issues/347

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 - Add gem [rack-mini-profiler](https://github.com/MiniProfiler/rack-mini-profiler) for displaying speed badge for every html page.
 - Add configuration for [NewRelic](https://devcenter.heroku.com/articles/newrelic) Heroku-addon
 - Implement the dynamic database name
+- Update [rollbar gem](https://github.com/rollbar/rollbar-gem) version up to 2.3.0
```
Comment (timurvafin) [inline CHANGELOG.md]: Please put this line on top of the list

File: Gemfile [hunk 2]
```diff
@@ -35,7 +35,7 @@ gem "pundit"
 gem "rack-mini-profiler"
 gem "rack-canonical-host"
 gem "responders"
-gem "rollbar", "~> 0.10.3"
+gem "rollbar", "~> 2.3.0"
```
Comment (timurvafin) [inline Gemfile]: Let's remove version

**Comment by dmitrytrager** [comment]
:ok:

**Comment by timurvafin** [comment]
Please cleanup commits and we will be ready to merge.

**Comment by ruslankhaertdinov** [comment]
@timurvafin done

**Pull request merged**